### PR TITLE
KLYNTAR listing

### PR DESCRIPTION
**Pull request by VladChernenko**
<div align="center">

### KLYNTAR - nextgen, "cloud-of-everything" project that relies on decentralization & security & TPS of all blockchains in the industry, has multilevel sharding, multitasking, EVM & WASM & Docker compatibility, post-quantum cryptography, RWX codeless smart-contracts, etc.

<a href="https://github.com/KLYN74R">🔥Follow us!🔥</a>

</div>

### CODE REVIEW

**Comment by VladChernenko** [comment]
Added more commits after Prettier execution

File: _data/chains/eip155-7331.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://klyntar.org",
+  "shortName": "kly",
+  "chainId": 7331,
+  "networkId": 7331,
+  "icon": "klyntar",
+  "explorers": [
+    {
+      "name": "klyntarscan",
+      "url": "https://klyntarscan.org",
```
Comment (ligi) [inline _data/chains/eip155-7331.json:23]: this URL is not reachable

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
explorer is still not reachable - also if this is a incubating chain add status incubating

**Comment by VladChernenko** [comment]
@ligi Decided to remove explorer URL and add a status

**Pull request merged**